Project.toml

```diff
@@ -4,13 +4,21 @@ repo = "https://github.com/JuliaDiff/TaylorSeries.jl.git"
 version = "0.16.0"
 
 [deps]
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
```

Comment:
Question: Why do we need InteractiveUtils? (I couldn't find where it was used...)

Reply:
Thanks, it was a leftover from some local tests, it's now removed